corona-warn-app/cwa-app-android
# More human readable server errors (EXPOSUREAPP-7136)

### Testing
1. add pending PCR/RAT test
1. turn off wifi
1. restart app
2.  see the error message
